Project-I-Social-Card/social-card/src/components/HeaderComponents/HeaderContent.js

```diff
@@ -1,2 +1,17 @@
-import React from 'react';
-import './Header.css';
+import React from "react";
+import "./Header.css";
+
+function HeaderContent() {
+  return (
+    <div className="headerContent">
+      <p className="summary">
+        Let's learn React by building simple interfaces with components. Don't
```

Comment:
again, try to pass all of this as props for practice, get into prop drilling down through 2 components, it will become your best friend until redux.